base/managers.jl

```diff
@@ -179,10 +179,10 @@ function launch_on_machine(manager::SSHManager, machine, cnt, params, launched,
 
     # the default worker timeout
     tval = haskey(ENV, "JULIA_WORKER_TIMEOUT") ?
-        `export JULIA_WORKER_TIMEOUT=$(ENV["JULIA_WORKER_TIMEOUT"]);` : ``
+        `export JULIA_WORKER_TIMEOUT=$(ENV["JULIA_WORKER_TIMEOUT"])\;` : ``
 
     # Julia process with passed in command line flag arguments
-    cmd = `cd $dir && $tval $exename $exeflags`
+    cmd = `cd $dir '&&' $tval $exename $exeflags`
```

Comment:
If you wish to test, set `JULIA_TESTFULL=1` in your env. Enable password less ssh login into your local machine. Run the parallel test. This will test SSH Manager.

Else I can test this locally in a couple of days.